### PR TITLE
[TH2-1163] Added parameters to CradleManager.init() to define maximum batch sizes.

### PR DESCRIPTION
**Pull request by Vladimir-Panarin-Exactpro**

### CODE REVIEW

File: cradle-core/src/main/java/com/exactpro/cradle/CradleManager.java [hunk 1]
```diff
@@ -30,20 +32,25 @@ public CradleManager()
...
 	 * Initializes manager to get access to Cradle storage using given name of data instance
 	 * @param instanceName name of data instance
 	 * @param prepareStorage flag that indicates if underlying storage on disk can be created or its structure can be updated, if needed
+	 * @param maxMessageBatchSize maximum size of {@link StoredMessageBatch} (in bytes) allowed for {@link CradleStorage} while writing data
+	 * @param maxTestEventBatchSize maximum size of {@link StoredTestEventBatch} (in bytes) allowed for {@link CradleStorage} while writing data
 	 * @throws CradleStorageException if access to Cradle storage cannot be established
 	 */
 	
-	public void init(String instanceName, boolean prepareStorage) throws CradleStorageException
+	public void init(String instanceName, boolean prepareStorage, long maxMessageBatchSize, long maxTestEventBatchSize) throws CradleStorageException
```
Comment (Nikita-Smirnov-Exactpro) [inline cradle-core/src/main/java/com/exactpro/cradle/CradleManager.java:51]: Please add the old overload: init(String instanceName, boolean prepareStorage) for backward compatibility

File: cradle-core/src/main/java/com/exactpro/cradle/CradleManager.java [hunk 2]
```diff
@@ -56,7 +63,7 @@ public void init(String instanceName, boolean prepareStorage) throws CradleStora
 	
 	public void init(String instanceName) throws CradleStorageException
 	{
-		initStart(instanceName, false);
+		initStart(instanceName, false, 0, 0);
```
Comment (Nikita-Smirnov-Exactpro) [inline cradle-core/src/main/java/com/exactpro/cradle/CradleManager.java:66]: Maybe use default values instead of `0, 0`

Comment (Vladimir-Panarin-Exactpro) [reply cradle-core/src/main/java/com/exactpro/cradle/CradleManager.java]: Agreed, done.

File: cradle-core/src/main/java/com/exactpro/cradle/CradleManager.java (hunk 1 above)
Comment (Vladimir-Panarin-Exactpro) [reply cradle-core/src/main/java/com/exactpro/cradle/CradleManager.java:51]: Done.

File: cradle-core/src/main/java/com/exactpro/cradle/CradleManager.java [hunk 3]
```diff
@@ -74,11 +94,13 @@ public void dispose() throws CradleStorageException
 	 * Starts initialization to get access to Cradle storage using given name of data instance
 	 * @param instanceName name of data instance
 	 * @param prepareStorage flag that indicates if underlying storage on disk can be created or its structure can be updated, if needed
+	 * @param maxMessageBatchSize maximum size of {@link StoredMessageBatch} (in bytes) allowed for {@link CradleStorage} while writing data
+	 * @param maxTestEventBatchSize maximum size of {@link StoredTestEventBatch} (in bytes) allowed for {@link CradleStorage} while writing data
 	 * @throws CradleStorageException if access to Cradle storage cannot be established
 	 */
-	public void initStart(String instanceName, boolean prepareStorage) throws CradleStorageException
+	public void initStart(String instanceName, boolean prepareStorage, long maxMessageBatchSize, long maxTestEventBatchSize) throws CradleStorageException
```
Comment (Nikita-Smirnov-Exactpro) [inline cradle-core/src/main/java/com/exactpro/cradle/CradleManager.java:101]: Please add the old overload: initStart(String instanceName, boolean prepareStorage) for backward compatibility

**Review by Nikita-Smirnov-Exactpro — APPROVED** [review]

**Pull request merged**